PC/bdist_wininst/install.c

```diff
@@ -1683,10 +1683,10 @@ SelectPythonDlgProc(HWND hwnd, UINT msg, WPARAM wParam, LPARAM lParam)
                                          &py_major, &py_minor);
                         if (result == 2) {
 #ifdef _DEBUG
-                            wsprintf(pythondll, "python%d%d_d.dll",
+                            wsprintf(pythondll, "python%d_%d_d.dll",
                                      py_major, py_minor);
 #else
-                            wsprintf(pythondll, "python%d%d.dll",
+                            wsprintf(pythondll, "python%d_%d.dll",
```

Comment:
This will cause us to have to manually rebuild and commit `wininst-...exe` in Lib/distutils/command, and just glancing at the code I can see in the PR this appears to have a list of Python installs, so we may need logic here to only insert the underscore for 3.10 and later (or we can get PEP 632 approved and then ignore this file completely)

Reply:
Even if PEP 632 to remove distutils is approved, these changes will need to be reflected in pypa/setuptools' copy of distutils, or did they drop bdist_wininst ?

Reply:
Question for @jaraco, but hopefully they dropped it.

Reply:
Thusfar, it only [raises a DeprecationWarning](https://github.com/pypa/distutils/blob/68f5b8181e1f42fd12586752d93e97bda1649d1d/distutils/command/bdist_wininst.py#L65-L67) (aka not a user-visible warning), so may need some care to disable it.

Reply:
I think it's fine to merge with or without this change, and we'll just track the distutils breakage as a new bug. Don't hold up this PR on it.